Tasks/Common/kubernetes-common-v2/kubernetesmanifestutility.ts

```diff
@@ -124,4 +156,16 @@ function isLoadBalancerIPAssigned(status: any) {
         return true;
     }
     return false;
+}
+
+function getEnvironmentUrl(): string {
+    const environmentId = tl.getVariable('Environment.Id');
+    let requestUrl = null;
+    if (environmentId) {
+        requestUrl = tl.getVariable('System.TeamFoundationCollectionUri') + tl.getVariable('System.TeamProject') + '/_environments/' + tl.getVariable('Environment.Id');
+        const resourceId = tl.getVariable('Environment.ResourceId');
+        requestUrl = resourceId ? requestUrl + '/providers/kubernetes/' + resourceId : requestUrl;
```

Comment:
Can this be done at server side? Where you dump a variable called `Environment.ResourceUrl` or something like that? 

@vithati thoughts?

Reply:
We can do that enhancement later.